lib/ClangImporter/ImportDecl.cpp

```diff
@@ -816,8 +816,8 @@ static bool isPrintLikeMethod(DeclName name, const DeclContext *dc) {
 using MirroredMethodEntry =
   std::tuple<const clang::ObjCMethodDecl*, ProtocolDecl*, bool /*isAsync*/>;
 
-ImportedType tryImportOptionsTypeForField(const clang::QualType type,
-                                          ClangImporter::Implementation &Impl) {
+ImportedType tryImportOptionsType(clang::QualType type,
```

Comment:
Can you say "option set"

Reply:
so `tryImportOptionSetType` ?

Reply:
Aye, ok so `findOptionSetType` Ok. will do